app/models/miq_server/update_management.rb

```diff
@@ -160,7 +162,7 @@ def apply_updates
     # MiqDatabase.cfme_package_name will update only the CFME package tree.  (Won't disturb the database)
     # "" will update everything
     packages_to_update = EvmDatabase.local? ? [MiqDatabase.cfme_package_name] : []
-    LinuxAdmin::Yum.update(*packages_to_update)
+    File.write(UPDATE_FILE, packages_to_update.join(","))
```

Comment:
I think ',' is a good separator.  Can rpms have commas?  I don't think so, right?